ema_workbench/examples/vensim_example.py

```diff
@@ -23,10 +23,12 @@
     # instantiate a model
     wd = r'./models/vensim example'
     vensimModel = VensimModel("simpleModel", wd=wd,
-                              model_file=r'\model.vpm')
+                              model_file=r'model.vpm') 
```

Comment:
the 'r' is not necessary here. It was not even needed in the old version of the code